### PR TITLE
HTML WEBSITE JOB PORTAL

### PR DESCRIPTION
**Pull request by sourabh513**

### CODE REVIEW

File: sourabh/WEBSITE/compains.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+					<input type="text"placeholder="Email...">
+					<textarea placeholder="Message"></textarea><br>
+					<button type="submit" class="btn btn-primary">Login</button>
+				</form>
+			</div>
+		</div>
+	</footer>
+
+</body>
+
```
Comment (onkar-bit) [inline sourabh/WEBSITE/compains.html]: remove extra space

File: sourabh/WEBSITE/index.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/index.html]: set proper indentation

File: sourabh/WEBSITE/index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+		</div>
+	</div>
+<!-- ................index content................... -->
+    <div class="image">
+	    <img src="indexbgimage.jpg">
+	    <div class="text">
+	       <h2>Build Your Career</h2>
+	    </div>
+    </div>	
+  			<div class="one">
```
Comment (onkar-bit) [inline sourabh/WEBSITE/index.html]: remove extra space

File: sourabh/WEBSITE/index.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+       <h2>Get your jobs</h2>
+       <p>Get your Popular jobs 202 New job</p>
+ 	</div>
+
```
Comment (onkar-bit) [inline sourabh/WEBSITE/index.html]: remove extra space and align properly

File: sourabh/WEBSITE/index.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+					<input type="text"placeholder="Your Name">
+					<input type="text"placeholder="Email...">
+					<textarea placeholder="Message"></textarea><br>
+					<button type="submit" class="btn btn-primary">Login</button>
+				</form>
+			</div>
+		</div>
+	</footer>
+</body>
+</html>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/index.html]: add new line

File: sourabh/WEBSITE/login [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/login]: need one tab space

File: sourabh/WEBSITE/login [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  <form>
+  <h2>Login</h2>
+  <label for="fname">User name:</label><br>
+  <input type="text" id="fname" name="fname" placeholder ="User name"><br>
+  <label for="fname">Password:</label><br>
+  <input type="text" id="fname" name="fname" placeholder ="Password"><br>
+  <input type="Login" value="Login">
+</form> 
+</body>
+</html>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/login]: add new line

File: sourabh/WEBSITE/login.html [hunk 8]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/login.html]: set proper indentation

File: sourabh/WEBSITE/login.html [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+			<li class="active"><a href="login.html">Login</a></li>
+			<li class="hr"><a href="aboutus.html">aboutus</a></li>
+			<li class="hr"><a href="terms & condition.html">terms & conditions</a></li>
+			<li style="float:right"><a href="#" style="padding: 0px"><img src="logo.png" alt="logo" class="logo"></a></li>
+			</ul>
+		</div>
+	</div>
+	<!-- sign up body -->
+        <div class="center">
+				<h4>Login</h4>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/login.html]: set proper indentation and remove extra space

File: sourabh/WEBSITE/services.html [hunk 10]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/services.html]: set proper indentation

File: sourabh/WEBSITE/services.html [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+						</div>
+					</div>
+				</div>
+			<div class="row">
+				<input type="button" class="btn brows-btn" value="Brows All Jobs" />
+			</div>
+		</div>
+	</section>
+</body>
+</html>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/services.html]: add new line

File: sourabh/WEBSITE/signup.html [hunk 12]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/signup.html]: set proper indentation in whole page

File: sourabh/WEBSITE/terms & condition.html [hunk 13]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (onkar-bit) [inline sourabh/WEBSITE/terms & condition.html]: set proper indentation in whole page

File: sourabh/WEBSITE/terms & condition.html [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+		body { 
+		  /*color: white;*/ 
+		  padding: 10px;
+		}
+		h1
+		{
+			  font-variant:;
+ 			 font-size: 90px;
+ 			 text-align: center; 
+  /*color: white;*/
```
Comment (onkar-bit) [inline sourabh/WEBSITE/terms & condition.html]: remove comment

File: sourabh/WEBSITE/Style.css [hunk 15]
```diff
@@ -0,0 +1,354 @@
...
+  font-family:poppins;
+  margin-top:10px;
+}
+.C1,.C2,.C3,.C4
+ {   
+   padding-left: 66px;
+  /* padding-top: 30px;*/
+ }
+
+/* login cssssss........................ */
```
Comment (SnehaMakade) [inline sourabh/WEBSITE/Style.css:354]: after colon i.e. after this(:) there should be space so add it

File: sourabh/WEBSITE/aboutus.html [hunk 16]
```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<title>Aboutus</title>
```
Comment (SnehaMakade) [inline sourabh/WEBSITE/aboutus.html:6]: add space in between aboutus make it as about us

File: sourabh/WEBSITE/aboutus.html [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+			.nav{
+				background-color: white;
+			}
+	</style>
+	</head>
+	<body>
+		<div class="nav">
+			<ul>
+				<li class="hr"><a href="index.html">Home</a></li>
+				<li class="hr"><a href="compains.html">compains</a></li>
```
Comment (SnehaMakade) [inline sourabh/WEBSITE/aboutus.html:46]: what is compains??

File: sourabh/WEBSITE/compains.html [hunk 18]
```diff
@@ -0,0 +1,126 @@
...
+			</ul>
+		</div>
+	<!-- First Grid: Logo & About -->
+		<div class="one">
+			<h2>Get your jobs</h2>
+			<p>Get your Popular jobs 202 New job</p>
+		</div>
+		<div class="two">
+			<h2>Find Popular Jobs</h2>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do</p>
```
Comment (SnehaMakade) [inline sourabh/WEBSITE/compains.html:26]: what is this?? write it in english only

**Pull request closed without merging**